### ISSUE TITLE
beerxml doesn't convert from C to F. 

### ISSUE THREAD

**Issue by jaywilliams916** [Manuel83/craftbeerpi#150]
When importing a beerxml, and your default temp unit is F, the temp will import without converting.

**Comment by Ph03n1X1** [comment]
I agree, this is a pain. I have to manually edit all my temps whenever I import.

**Comment by tmurph6** [comment]
Any resolution for this?  I imported beerxml yesterday to test, my CBP is set to F and my software is using F but upon import it showed mash temp as like 67.2F which is 153F converted to C.
